components/ca-tel.js

```diff
@@ -0,0 +1,175 @@
+define([
+    './helpers/create-element.js',
+    './helpers/fire-event.js',
+    './ca-select.js',
```

Comment:
Where it's a non-optional dependency, it seems to me to be an advantage of using a module loader to be able to do this so that the app only needs to declare what it actually wants to use in a black box way. But we need to make more dependencies optional with sensible fallbacks (esp. ca-form).